ch_3_AppDev_Basic/3_Quick_Start.ipynb

```diff
@@ -18,34 +18,43 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install bigdl-llm[all]"
+    "!pip install --pre --upgrade bigdl-llm[all]"
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This one-line command will install `bigdl-llm` with all the dependencies for common LLM application development.\n",
+    "This one-line command will install the latest `bigdl-llm` with all the dependencies for common LLM application development.\n",
     "\n",
     "\n",
     "## 3.2 Load a pretrained Model\n",
     "\n",
-    "Now let's load a relatively small LLM model, i.e [open_llama_3b_v2](https://huggingface.co/openlm-research/open_llama_3b_v2). \n",
+    "Now let's load a relatively small LLM model, i.e [open_llama_3b_v2](https://huggingface.co/openlm-research/open_llama_3b_v2) as an example.\n",
     "\n",
-    "Simply use one-line `transformers`-style API in `bigdl-llm` to load `open_llama_3b_v2` with INT4 optimization (by specifying `load_in_4bit=True`) as follows:\n"
+    "### 3.2.1 Optimize Model\n",
+    "You can load `open_llama_3b_v2` with [Hugging Face Transformers](https://huggingface.co/docs/transformers/index) and then just need to use one-line Pytorch-style API in `bigdl-llm` to accelerate `open_llama_3b_v2` with INT4 optimization as follows:\n"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from bigdl.llm.transformers import AutoModelForCausalLM\n",
+    "from transformers import LlamaForCausalLM, LlamaTokenizer\n",
+    "from bigdl.llm import optimize_model\n",
+    "\n",
+    "model_path = 'openlm-research/open_llama_3b_v2'\n",
     "\n",
-    "model = AutoModelForCausalLM.from_pretrained(pretrained_model_name_or_path=\"openlm-research/open_llama_3b_v2\",\n",
-    "                                             load_in_4bit=True)"
+    "tokenizer = LlamaTokenizer.from_pretrained(model_path)\n",
```

Comment:
`tokenizer = LlamaTokenizer.from_pretrained(model_path)` put tokenizer initialization to Run LLM section